analysis_driver/driver.py

```diff
@@ -172,6 +173,15 @@ def demultiplexing_pipeline(dataset):
         app_logger.error('File not found: %s', conversion_xml)
         exit_status += 1
 
+    adapter_trim_file = os.path.join(fastq_dir, 'Stats', 'AdapterTrimming.txt')
+    if os.path.exists(adapter_trim_file):
+        app_logger.info('Found AdapterTrimming. Sending data.')
+        crawler = RunCrawler(run_id, sample_sheet, adapter_trim_file=adapter_trim_file)
+        crawler.send_data()
+    else:
+        app_logger.error('File not found: %s', adapter_trim_file)
+        exit_status += 1
+
```

Comment:
In line 167 above, because the 3rd argument is now `adapter_trim_file`, `conversion_xml` will be passed as an adaptor trim file. There are also two almost-identical blocks here - can't we merge them? Something like:

```python
if os.path.exists(conversion_xml) and if os.path.exists(adaptor_trim_file):
   app_logger.info('Found ConversionStats and AdaptorTrimming. Sending data.')
    c = RunCrawler(run_id, sample_sheet, adaptor_trim_file, conversion_xml, run_dir=fastq_dir)
    c.send_data()
else:
    app_logger.error('ConversionStats or AdaptorTrimming not found.')
    exit_status += 1
```